packages/netsuite-adapter/test/client/graph_utils.test.ts

```diff
@@ -32,7 +32,7 @@ describe('graph utils tests', () => {
 
 
   it('should find the nodes dependencies', async () => {
-    expect(testGraph.getNodeDependencies(testNode1)).toEqual([testNode1, testNode2, testNode3])
+    expect(testGraph.getNodeDependencies(testNode1)).toEqual([testNode2, testNode3, testNode1])
```

Comment:
nit - because the order isn't important
```suggestion
    const res = testGraph.getNodeDependencies(testNode1)
    expect(res).toHaveLength(3)
    expect(res).toEqual(expect.arrayContaining([testNode1, testNode2, testNode3]))
```